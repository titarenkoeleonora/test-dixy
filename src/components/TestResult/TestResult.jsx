import React from 'react';

const TestResult = (props) => {
  const {result} = props;

  return (
    <div className="test-result form-test__result">
      <p className="test-result__title">{result.title}</p>
        <p className="test-result__text">
          {result.text}
        </p>
    </div>
  );
};

export default TestResult;