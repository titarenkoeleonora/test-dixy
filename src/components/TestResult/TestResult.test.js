import TestResult from './TestResult';
import renderer from 'react-test-renderer';

const testResults = {
  id: 'a',
  title: 'НЕДОВЕР НЕДОВЕРЫЧ',
  text: 'Ты знаешь марку самой натуральной колбасы, а также где максимально выгодно ее купить. Твой список покупок всегда наготове, и ты умело им распоряжаешься, не теряя голову при виде акционных товаров. Мудрость и рациональность в продуктовом шопинге – твоя отличительная черта, это очень ценные качества, береги их.',
};


it(`Correctly TestResult render`, () => {
  const tree = renderer
    .create(
      <TestResult
        result={testResults}
      />
    );

  expect(tree).toMatchSnapshot();
});
