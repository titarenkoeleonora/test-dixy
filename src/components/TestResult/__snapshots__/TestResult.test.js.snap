// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly TestResult render 1`] = `
<div
  className="test-result form-test__result"
>
  <p
    className="test-result__title"
  >
    НЕДОВЕР НЕДОВЕРЫЧ
  </p>
  <p
    className="test-result__text"
  >
    Ты знаешь марку самой натуральной колбасы, а также где максимально выгодно ее купить. Твой список покупок всегда наготове, и ты умело им распоряжаешься, не теряя голову при виде акционных товаров. Мудрость и рациональность в продуктовом шопинге – твоя отличительная черта, это очень ценные качества, береги их.
  </p>
</div>
`;
